src/streamlink/plugins/beattv.py

```diff
@@ -1,324 +1,48 @@
-                self.reader.buffer.write(data)
-                if self.closed:
-                    return
+from streamlink.stream import HLSStream, HTTPStream
+from streamlink.utils import parse_json, search_dict
 
-            self.logger.debug("Download of chunk {0}/{1}/{2} complete",
-                              chunk.recording,
-                              chunk.quality,
-                              chunk.sequence)
-        except IOError as err:
-            self.logger.error("Failed to read chunk {0}/{1}/{2}: {3}",
-                              chunk.recording,
-                              chunk.quality,
-                              chunk.sequence, err)
-
-
-class BeatStreamWorker(SegmentedStreamWorker):
-    def __init__(self, *args, **kwargs):
-        SegmentedStreamWorker.__init__(self, *args, **kwargs)
-
-    def iter_segments(self):
-        quality = QUALITY_MAP[self.stream.quality]
-        for part in self.stream.parts:
-            duration = part["duration"]
-            if not part.get("recording"):
-                recording = part["id"]
-                extension = "part"
-            else:
-                recording = part["recording"]
-                extension = "rec"
-
-            chunks = int(duration / 12) + 1
-            start = int(part.get("start", 0) / 12)
-            for sequence in range(start, chunks + start):
-                if self.closed:
-                    return
-
-                self.logger.debug("Adding chunk {0}/{1}/{2} to queue",
-                                  recording, quality, sequence)
-
-                yield Chunk(recording, quality, sequence, extension)
-
-
-class BeatStreamReader(SegmentedStreamReader):
-    __worker__ = BeatStreamWorker
-    __writer__ = BeatStreamWriter
-
-    def __init__(self, stream, *args, **kwargs):
-        self.logger = logging.getLogger("streamlink.stream.beat")
-
-        SegmentedStreamReader.__init__(self, stream, *args, **kwargs)
-
-
-class BeatStream(Stream):
-    __shortname__ = "beat"
-
-    def __init__(self, session, parts, quality):
-        Stream.__init__(self, session)
-
-        self.parts = parts
-        self.quality = quality
-
-    def __repr__(self):
-        return ("<BeatStream({0!r}, {1!r}>").format(len(self.parts),
-                                                    self.quality)
-
-    def __json__(self):
-        return dict(parts=self.parts, quality=self.quality,
-                    **Stream.__json__(self))
-
-    def open(self):
-        if not CAN_DECRYPT:
-            raise StreamError(
-                "pyCrypto or pycryptodome needs to be installed to decrypt this stream"
-            )
-
-        reader = BeatStreamReader(self)
-        reader.open()
-
-        return reader
+log = logging.getLogger(__name__)
 
 
 class BeatTV(Plugin):
-    @classmethod
-    def can_handle_url(self, url):
-        return _url_re.match(url)
+    _url_re = re.compile(r"https?://(\w+\.)?be-at.tv/videos/.+")
```

Comment:
there is also `/video/`

https://www.be-at.tv/video/adam_beyer_hyte_nye_germany_2018